app/config/config_test.yml

```diff
@@ -1,5 +1,6 @@
 imports:
     - { resource: config_dev.yml }
+    - { resource: security_test.yml }
```

Comment:
what I mean: this runs the risk of deviating from the prod version, with all possible ramifications